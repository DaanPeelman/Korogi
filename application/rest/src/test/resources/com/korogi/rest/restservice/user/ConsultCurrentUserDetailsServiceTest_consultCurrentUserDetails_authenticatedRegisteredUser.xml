<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <USERS  ID="1"
            PROVIDER_ID="1"
            EMAIL="john.doe@test.com"
            USERNAME="john.doe"
            BIOGRAPHY="John Doe's Biography"
            ACTIVATED="true"
            SALT="1"

            CREATION_DATE="2016-08-27 00:00:00"
            CREATED_BY="john.doe"
            MODIFICATION_DATE="2016-08-27 00:00:00"
            MODIFIED_BY="john.doe"
            VERSION="1" />
</dataset>