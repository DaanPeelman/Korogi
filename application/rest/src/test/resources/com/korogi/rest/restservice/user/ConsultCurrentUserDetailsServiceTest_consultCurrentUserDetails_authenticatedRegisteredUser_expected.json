{
  "username": "john.doe",
  "biography": "John Doe's Biography",
  "links": []
}